test/e2e/e2e_test.go

```diff
@@ -313,3 +313,31 @@ func TestGCPBroker(t *testing.T) {
 	defer cancel()
 	GCPBrokerTestImpl(t, authConfig)
 }
+
+// TestCloudPubSubSourceWithGCPBroker tests we can knock a Knative Service from a GCPBroker from a CloudPubSubSource.
+func TestCloudPubSubSourceWithGCPBroker(t *testing.T) {
+	cancel := logstream.Start(t)
+	defer cancel()
```

Comment:
Just `defer logstream.Start(t)`?

Reply:
Does 
`cancel := logstream.Start(t)
 defer cancel()`
equal to 
`defer logstream.Start(t)`?

Reply:
Ah, I misread this.